### PR TITLE
Add missing forum acl migrations

### LINKED ISSUE #605: Missing phpbb tables?
Some of the phpbb tables seem to be missing from the migrations, giving you errors if you try to make new topics etc.

Tables that I've noticed missing: phpbb_acl_options, phpbb_acl_roles_data.

Do these need to be added to migrations and seeded at all? Or am I being retarded?

### PR DESCRIPTION
**Pull request by LiquidPL**
Fixes #605.

### CODE REVIEW

**Comment by nekodex** [comment]
Does this need seed data to go with it to work?

**Comment by LiquidPL** [comment]
Alright, now seeders set up permissions for the seeded users and forums.

However, this is not the case for any user that is created manually, as in that case an entry in `phpbb_user_group`. Should this be noted in the readme?

**Comment by peppy** [comment]
When creating a user they should be assigned to the default group, which gets some amount of ACL permissions, so I don't think it's necessary to note it in the readme for now.

**Comment by peppy** [comment]
Awarding 40 bounty for this.

File: app/Models/Forum/AuthGroup.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace App\Models\Forum;
+
+use Illuminate\Database\Eloquent\Model;
+
+class AuthGroup extends Model
+{
+    protected $table = 'phpbb_acl_groups';
```
Comment (nanaya) [inline app/Models/Forum/AuthGroup.php]: um, this one is also defined for model in `app/Models/Forum/Authorize.php`

**Pull request merged**